03LessonThree/homework/git-scm.html

```diff
@@ -52,7 +52,7 @@
             </div>
         </div>
     </div>
-    <div id="footer">
+    <footer>
         This open sourced site is hosted on GitHub.<br/> Patches, suggestions and comments are welcome.
     </div>
 </body>

```

Comment:
You understood what to do for the assignment but you need to also close your semantic HTML tags.